testing/opae-c/test_buffer_c.cpp

```diff
@@ -54,8 +54,12 @@ class buffer_c_p : public mock_opae_p<2> {
     ASSERT_EQ(fpgaInitialize(NULL), FPGA_OK);
     ASSERT_EQ(fpgaGetProperties(nullptr, &filter_), FPGA_OK);
     ASSERT_EQ(fpgaPropertiesSetObjectType(filter_, FPGA_ACCELERATOR), FPGA_OK);
-    ASSERT_EQ(fpgaPropertiesSetDeviceID(filter_,
-              platform_.devices[0].device_id), FPGA_OK);
+    auto device_id = platform_.devices[0].device_id;
+    if (platform_.devices[0].num_vfs) {
+      device_id++;
```

Comment:
ie, 0xbcc0 become 0xbcc1.. but this behavior is not guaranteed I think.  I'm not sure this will always be the case.

Reply:
You're right. This will work for now but fixing this the right way would require more work to the testing hw tables and data structures.